public/checkout.js

```diff
@@ -41,7 +41,8 @@ var packeteryLoadCheckout = function( $, settings ) {
 		var getShippingRateId = function() {
 			var $selectedRadio = $( '#shipping_method input[type="radio"]:checked' );
 			if ( $selectedRadio.length ) {
-				return $selectedRadio.val();
+				var radioValue = $selectedRadio.val();
+				return radioValue.replace('packetery_shipping_method:','');
```

Comment:
Zkusil bych to vyřešit tak, abys nemusel v js použít "packetery_shipping_method".  Toto označení přepravní  metody Zásilkovny máme v php jako konstantu a pokud bychom ji někdy změnili, musíme to změnit i zde. Tomu by jsme se chtěli vyhnout.